javascript/android-app-test.js

```diff
@@ -27,6 +27,11 @@ const desiredCaps = {
 
 let driver
 
+if (!username || !apiKey) {
+  console.log('Error: KOBITON_ACCESS_USERNAME and KOBITON_ACCESS_API_KEY are required to execute script')
```

Comment:
`Environment variables KOBITON_USERNAME and KOBITON_API_KEY are required to execute script`